config/privateLinks.js

```diff
@@ -0,0 +1,65 @@
+const { JSDOM } = require("jsdom");
+
+const privateUrlSlugs = [
+  "18f.slack.com",
+  "anywhere.gsa.gov",
+  "bookit.gsa.gov",
+  "calendar.gsa.gov",
+  "connect.gsa.gov",
+  "docs.google.com",
+  "drive.google.com",
+  "ea.gsa.gov",
+  "email.gsa.gov",
+  "eopf.opm.gov",
+  "gcims.gsa.gov",
+  "github.com/18F/Accessibility_Reviews",
+  "github.com/18F/blog-drafts",
+  "github.com/18F/codereviews",
+  "github.com/18F/DevOps",
+  "github.com/18F/Infrastructure",
+  "github.com/18F/security-incidents",
+  "github.com/18F/staffing",
+  "github.com/18F/team-api.18f.gov",
+  "github.com/18F/writing-lab",
+  "gkey.gsa.gov",
+  "gsa-tts.slack.com",
+  "gsa.my.salesforce.com",
+  "gsaolu.gsa.gov",
+  "hrlinks.gsa.gov",
+  "hrprod.hr.gsa.gov",
+  "insite.gsa.gov",
+  "mail.gsa.gov",
+  "meet.gsa.gov",
+  "sign.gsa.gov",
+  "tock.18f.gov",
+];
+
+module.exports = (eleventyConfig) => {
+  eleventyConfig.addTransform("private_links", async function (content) {
+    if (
+      this.inputPath !== "./pages/redirect.html" &&
+      this.outputPath.endsWith(".html")
```

Comment:
To clarify, this is saying that when content is being transformed into HTML but isn't a redirect page, run this find-and-replace?

Reply:
Yes! In #3609, I added comments to this transform to explain it more. I can pull those comments over into this PR to make it clearer, but I wanted to be careful about making this one even bigger than it already is.